packages/controllers/organization_detail.coffee

```diff
@@ -67,3 +103,15 @@ if Meteor.isServer
       organization = Organizations.findOne(orgId)
       if organization.userIsAdmin(@userId)
         organization.removeAdmin(profileId)
+
+    updateName: (name, id) ->
```

Comment:
Meteor has a single namespace for methods so these might need to be more specific.